lib/components/gene/__tests__/__snapshots__/about-test.js.snap

```diff
@@ -0,0 +1,197 @@
+exports[`test renders correctly 1`] = `
+<View>
+  <View>
+    <View
+      style={
+        Object {
+          "marginLeft": 0,
+          "marginRight": 0,
+        }
+      }>
+      <Text
+        accessible={true}
+        allowFontScaling={true}
+        ellipsizeMode="tail"
+        numberOfLines={0}
+        style={
+          Array [
+            Object {
+              "fontSize": 17,
+            },
+            Object {
+              "fontSize": 16,
+              "lineHeight": 20,
+              "marginBottom": 20,
+            },
+            Object {
+              "fontFamily": "AGaramondPro-Regular",
+            },
+          ]
+        }>
+        This is some example text
+      </Text>
```

Comment:
this is the rendered content for the gene about tab

![screen shot 2016-11-28 at 16 53 12](https://cloud.githubusercontent.com/assets/49038/20677490/2aa4aa80-b58b-11e6-802d-99d20a546d65.png)

basically nothing on this one, this file represents all the tests in the test file, so it's not 1 test 1 file.